From 0506e46640989e1f919388ad36890bd6063cd43a Mon Sep 17 00:00:00 2001
From: Zach van Rijn <me@zv.io>
Date: Sat, 26 Aug 2017 01:32:56 -0400
Subject: [PATCH] explicitly include <string.h> for memset()

[Thomas: taken from
https://github.com/boostorg/fiber/pull/142/commits/0506e46640989e1f919388ad36890bd6063cd43a. It
is not going to be applied upstream because the actual bug is in musl,
but it's a good enough workaround for now.]
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 libs/fiber/src/numa/linux/pin_thread.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libs/fiber/src/numa/linux/pin_thread.cpp b/libs/fiber/src/numa/linux/pin_thread.cpp
index ae696be3..e9b3c423 100644
--- a/libs/fiber/src/numa/linux/pin_thread.cpp
+++ b/libs/fiber/src/numa/linux/pin_thread.cpp
@@ -9,6 +9,7 @@
 extern "C" {
 #include <pthread.h>
 #include <sched.h>
+#include <string.h>
 }
 
 #include <system_error>
